samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -91,7 +91,7 @@ CamResolutionsTest.prototype = {
     var call = new Call();
     call.pc1.addStream(stream);
     call.establishConnection();
-    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 100);
+    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 1000);
```

Comment:
Btw I was thinking that due to timing one will likely only gather 4 samples during the 5 seconds... but I guess that is fine for now.